draft-demarco-oauth-status-attestations.md

```diff
@@ -342,25 +342,25 @@ encoding, for better readability:
 ~~~
 
 As exemplified for the JWT format, below is a non-normative example of a Credential proof of possession in CWT format. The CWT headers and payload are presented without applying signature and encoding for improved readability:
```

Comment:
please mention that the prosed non normative example is provided in an unencoded diagnostic representation ...

Reply:
The commit that aims to resolve this is: https://github.com/peppelinux/draft-demarco-oauth-status-attestations/pull/37/commits/5980f052b048e7d7cc6aad338e7f6386646e00e5